app/views/templates/_article.html.erb

```diff
@@ -2,26 +2,58 @@
   <%= @cms_page.label %>
 <% end %>
 
-<div class="row column">
-  <h1><%= @cms_page.label %></h1>
+<div class="template--article template-padding site-padding">
+  <div class="row">
+    <div class="small-12 medium-10 medium-centered large-8 column">
+      <%= link_to 'News & Documents', "#{@cms_site.path}/news-and-documents", title: 'Visit the News and Documents page', class: 'button--arrow' %>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-12 medium-10 medium-centered large-8 column">
+      <span class="article__date"><%= @cms_page.created_at.strftime('%d %B %Y') %></span>
 
-  <div>
-  <%= cms_block_content(:introduction) %>
+      <% unless cms_block_content(:article_type).strip.empty? %> 
+        <span class="article__category"><%= cms_block_content(:article_type) %></span>
+      <% end %>
+    </div>
   </div>
+  
+  <div class="row">
+    <div class="small-12 medium-10 medium-centered large-8 column article__main">
+      <h1><%= @cms_page.label %></h1>
+      
+      <% unless cms_block_content(:introduction).strip.empty? %> 
+        <div class="text--18 text--fira-sans">
+          <%= cms_block_content(:introduction) %>
+        </div>
+      <% end %>
+      
+      <% unless cms_block_content(:image).to_s.strip.empty? %>
```

Comment:
I have updated it to match what you have written for the other files. Can you explain what the & does here?